configs/flowable.json

```diff
@@ -177,25 +177,20 @@
     "https://documentation.flowable.com/sitemap.xml"
   ],
   "selectors": {
-    "lvl0": {
-      "selector": "h3.title",
-      "global": true,
-      "default_value": "Documentation"
-    },
-    "lvl1": {
-      "selector": "[aria-label='breadcrumbs'] li:nth-child(2) > a",
-      "global": true,
-      "default_value": "Documentation"
-    },
-    "lvl2": ".doc h1",
-    "lvl3": ".doc h2",
-    "lvl4": ".doc h3",
-    "lvl5": ".doc h4",
-    "lvl6": ".doc h5",
-    "text": ".doc p, .doc li",
-    "version": {
-      "selector": "span.version",
-      "global": true
+    "default": {
```

Comment:
Do not use a default selector_key if none of them are defined

Reply:
Thanks for your help.